### ISSUE TITLE
Leaks

### ISSUE THREAD

**Issue by mbjones** [NCEAS/metacat#855]
---

Author Name: **Brendan Hahn** (Brendan Hahn)
Original Redmine Issue: 5835, https://projects.ecoinformatics.org/ecoinfo/issues/5835
Original Date: 2013-01-30
Original Assignee: ben leinfelder

---

There are significant memory leaks that will eventually hang or kill the application server if metacat is restarted without terminating the entire server process. E.g. use of log4j dynamic configuration leaves orphan threads about (not really metacat's fault -- I'm amazed this is *still* a problem). Other threads, classloaders, etc along with the associated resources get left behind as well.

In short: should clean up better.

**Comment by mbjones** [comment]
---

Original Redmine Comment
Author Name: **Redmine Admin** (Redmine Admin)
Original Date: 2013-03-27T21:31:49Z

---

Original Bugzilla ID was 5835
